examples/tutorials/rigid_object_primitive_tutorial.py

```diff
@@ -0,0 +1,198 @@
+# [setup]
+import math
+import os
+from pathlib import Path
+
+import cv2
```

Comment:
I know this isn't specific to your tutorial (same thing happens in rigid_body_tutorial), but we don't list opencv in our requirements.txt. 

In that tutorial, we ask people in the webpage to install cv2. Should we make it a requirement or is that too much?

Reply:
It's not as bad it used to be to install opencv with the pip wheel packages being available, but we can pretty easily move away from it. I wouldn't recommend using the cv2.video_writer anyhow since imageio-ffmpeg is a much lighter wrapper around that anyhow and we already have that in our --all requirements. (I think we have cv2 in our --all requirements for habitat-api anyhow.